lib/text/ui_text_displayer.js

```diff
@@ -504,6 +508,19 @@ shaka.text.UITextDisplayer = class {
       regionElement.style.left = region.viewportAnchorX -
         region.regionAnchorX * region.width / 100 + viewportAnchorUnit;
```

Comment:
while the below is probably fine, in trying to figure out what was going on I realized that this and section uses region.height/width but doesn't account for whether it's in line units of precentages. This ends up working out for `left`, but for `top` we end up with `95%` but it should be `77.68%`, which is what we'd get if we multiply the height by the then lineHeightMultiple. We potentially should store the calculated height and width from above in a var so that we can use it here and below, if we wanted to keep that. Though, potentially shouldn't be necessary.

Reply:
@gkatsev Can you create a PR with it? Thanks!